cloudify_deployment_proxy/polling.py

```diff
@@ -150,7 +154,10 @@ def dep_logs_redirect(_client, execution_id):
 def dep_system_workflows_finished(_client, _check_all_in_deployment=False):
 
     try:
-        _execs = _client.executions.list(include_system_workflows=True)
+        _execs = _client.executions.list(
+            include_system_workflows=True,
+            _offset=PAGINATION_OFFSET,
```

Comment:
What do you think provide such as inputs to deployment proxy inputs or filter out all finished executions on manager side, e.g. started_at before today ?

Reply:
I made a new change, can you review? @0lvin

Reply:
So, I prefer not to start adding arbitrary inputs to operations. What I did was to [search for this constant in environment vars](https://github.com/cloudify-incubator/cloudify-utilities-plugin/pull/140/files#diff-45e1edee1b96d444ea2bc36aadbabd13R15).